geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -1738,6 +1769,37 @@ public Builder setPort(final Integer port) {
       return this;
     }
 
+    /**
+     * Sets the IP address to be used for membership-related traffic binding.
+     *
+     * @param membershipBindAddress a String containing the IP address to be used for
+     *        membership-related traffic binding.
+     * @return this Builder instance.
+     * @see #getMembershipBindAddress()
+     */
+    public Builder setMembershipBindAddress(final String membershipBindAddress) {
+      this.membershipBindAddress = membershipBindAddress;
+      return this;
+    }
+
+    /**
+     * Gets the IP address to be used for membership-related traffic binding.
+     *
+     * @return a String containing the IP address to be used for membership-related traffic binding.
+     */
+    public String getMembershipBindAddress() {
+      return this.membershipBindAddress;
+    }
+
+    /**
+     * Determines whether the membership-bind-address property is defined or not.
+     *
+     * @return a boolean value indicating if the membership-bind-address property is defined or not.
+     */
+    public boolean membershipBindAddressSpecified() {
```

Comment:
Since this is a User API, I recommend staying consistent with the other Builder methods, such as `isBindAddressSpecified()`, which would make this one `isMembershipBindAddressSpecified()`.

Any class or inner-class that is in a package without the word `internal` is a User API (unless the class or methods are package-private).